packages/ui-extensions/buildTargetDts.ts

```diff
@@ -73,13 +73,11 @@ function createInitialTargetDefinition({
   const targetPath = join(directory, fileName);
 
   const template = `import type {ExtensionTargets} from '../extension-targets';
+  ${surface === 'customer-account' ? `import '../globals';\n` : ''}
```

Comment:
are we the only surface with global APIs?

Reply:
We could update to do a check before attempting to import the global. I think most surfaces will have globals eventually

Reply:
yeah we are the only ones for now, once other surfaces implement their globals they can remove this check and import `...globals` too